cplusplus.json

```diff
@@ -953,5 +953,52 @@
   "imports": ["Var"], "examples": ["https://github.com/szi/vartypes/tree/master/example/src"]},
   {"id": "cpp.zbar", "name": "Zbar", "tech": ["images"], "tags": ["barcode-scanner"], "repo": "https://sourceforge.net/p/zbar/code/ci/default/tree/",
-  "imports": ["zbar.h"], "examples": ["https://sourceforge.net/p/zbar/code/ci/default/tree/examples/"]}
+  "imports": ["zbar.h"], "examples": ["https://sourceforge.net/p/zbar/code/ci/default/tree/examples/"]},
+
+  {"id": "cpp.abseil-cpp", "name": "Abseil-C++", "tech": ["app-arch"], "tags": ["stl-extension"], "repo": "abseil/abseil-cpp",
+  "imports": ["absl/"], "examples": ["https://github.com/abseil/abseil-cpp/tree/master/absl"]},
+  {"id": "cpp.apr", "name": "APR", "tech": ["app-arch"], "tags": ["cross-platform", "run-time"], "repo": "http://svn.apache.org/repos/asf/apr/apr/trunk/",
+  "imports": ["apr.h", "apr_"], "examples": ["http://svn.apache.org/repos/asf/apr/apr/trunk/test/"]},
+  {"id": "cpp.bde", "name": "BDE", "tech": ["app-arch"], "tags": ["stl"], "repo": "bloomberg/bde",
+  "imports": ["bsl_", "bslma_"], "examples": ["https://github.com/bloomberg/bde/tree/master/samples"]},
+  {"id": "cpp.cxxomfort", "name": "cxxomfort", "tech": ["utils"], "tags": ["backporting"], "repo": "http://ryan.gulix.cl/fossil.cgi/cxxomfort/dir?ci=0c1dc33fc412517f&name=cxxomfort/cxxomfort",
+  "imports": ["cxxomfort/"], "examples": ["https://github.com/lmachucab/cxxomfort-examples/tree/master/examples"]},
+  {"id": "cpp.eastl", "name": "EASTL", "tech": ["app-arch"], "tags": ["high-performance" ,"stl"], "repo": "electronicarts/EASTL",
+  "imports": ["EASTL/"], "examples": ["https://github.com/electronicarts/EASTL/tree/master/benchmark"]},
+  {"id": "cpp.ffead-cpp", "name": "ffead-cpp", "tech": ["cpp-web"], "tags": ["framework"], "repo": "sumeetchhetri/ffead-cpp",
+  "imports": [], "examples": ["https://github.com/sumeetchhetri/ffead-cpp/tree/master/tests"]},
+  {"id": "cpp.functional-plus", "name": "FunctionalPlus", "tech": ["functional"], "tags": ["readable-code"], "repo": "Dobiasd/FunctionalPlus",
+  "imports": ["fplus/"], "examples": ["https://github.com/Dobiasd/FunctionalPlus/tree/master/examples"]},
+  {"id": "cpp.glib", "name": "GLib", "tech": ["app-arch"], "tags": ["core-utils"], "repo": "https://gitlab.gnome.org/GNOME/glib/tree/master",
+  "imports": ["glib.h", "glib-"], "examples": ["https://gitlab.gnome.org/GNOME/glib/tree/master/tests"]},
+  {"id": "cpp.juce", "name": "JUCE", "tech": ["audio"], "tags": ["cross-platform"], "repo": "WeAreROLI/JUCE",
+  "imports": ["juce_"], "examples": ["https://github.com/WeAreROLI/JUCE/tree/master/examples"]},
+  {"id": "cpp.libphenom", "name": "libPhenom", "tech": ["app-arch"], "tags": ["eventing"], "repo": "facebook/libphenom",
+  "imports": ["phenom/"], "examples": ["https://github.com/facebook/libphenom/tree/master/examples"]},
+  {"id": "cpp.libsourcey", "name": "LibSourcey", "tech": ["networking"], "tags": ["evented"], "repo": "sourcey/libsourcey",
```

Comment:
okay, I thought that "eventing" was a funny word, but "evented" is really funny :). Let's do tags==p2p-streaming instead